python/fusion_engine_client/analysis/analyzer.py

```diff
@@ -1131,11 +1145,14 @@ def _plot_wheel_ticks_or_speeds(self, type):
             params = copy.deepcopy(self.params)
             params['max_messages'] = 2
             dt_sec = None
-            result = self.reader.read(message_types=wheel_measurement_type, remove_nan_times=False, **params)
-            data = result[wheel_measurement_type.MESSAGE_TYPE]
-            if len(data.measurement_time) == 2:
-                dt_sec = data.measurement_time[1] - data.measurement_time[0]
-            else:
+
+            if wheel_measurement_type is not None:
+                result = self.reader.read(message_types=wheel_measurement_type, remove_nan_times=False, **params)
+                data = result[wheel_measurement_type.MESSAGE_TYPE]
+                if len(data.measurement_time) == 2:
+                    dt_sec = data.measurement_time[1] - data.measurement_time[0]
+
+            if dt_sec is None and vehicle_measurement_type is not None:
```

Comment:
So I understand, it seems like `dt_sec` is just use to check whether this plot should be suppressed (The warning it generates seems to be misleading since the file size isn't used as a precondition). It would seem more consistent to also calculate it for  vehicle_measurement_type of both are present and take the max?

Reply:
Ah yeah this should be conditioned on `self.long_log_detected`.

It does respect the `--truncate=false` -- this stuff is all nested under an `if self.truncate_data:`

I don't think we really need the max. In practice, logs should only have wheel or vehicle speed. We support both just on the VERY rare chance that both are present.